lib/spdy/server.js

```diff
@@ -219,15 +219,16 @@ proto.emit = function emit(event, req, res) {
 
   if (!(req.socket._handle instanceof spdy.handle)) {
     debug('not spdy req/res');
-    req.isSpdy = false;
-    req.spdyVersion = 1;
+    req.isSpdy = res.isSpdy = false;
```

Comment:
Let's split this and the next ones into two lines, if you don't have a strong objection to this.